org.eclipse.lsp4e/src/org/eclipse/lsp4e/LanguageServers.java

```diff
@@ -268,6 +282,72 @@ public long getStartVersion() {
 			return this.startVersion;
 		}
 
+		public SingleLanguageServerDocumentExecutor toExecutor(final @NonNull LanguageServerWrapper serverWrapper) {
```

Comment:
This I would also remove, by the way, and keep the API small. It is only a bit of syntactic sugar, and we should only add those if it really pays. I am not sure yet it does.

Reply:
Yeah, but it does make the api more convenient, especially as a lot of these methods will get used in lambdas. LSP4e is already somewhat rich in lambda functions that ramble on for multiple lines and are hard to read..!